trade-finance-manager-ui/public/js/enableSelectAllTableCheckbox.js

```diff
@@ -1,2 +1,35 @@
-var DTFS_TFM;!function(){var e=document.querySelector('table.govuk-table input[type="checkbox"]#select-all-checkbox'),c=document.querySelectorAll('table.govuk-table input[type="checkbox"]:not(#select-all-checkbox)'),t=function(){return Array.prototype.every.call(c,(function(e){return e.checked}))},n=function(){t()?e.checked=!0:Array.prototype.some.call(c,(function(e){return!e.checked}))&&(e.checked=!1)};e&&(e.addEventListener("change",(function(e){e.target.checked?c.forEach((function(e){e.checked=!0})):c.forEach((function(e){e.checked=!1}))})),t()&&(e.checked=!0),c.forEach((function(e){e.addEventListener("change",(function(){n()}))}))),(DTFS_TFM=void 0===DTFS_TFM?{}:DTFS_TFM).enableSelectAllTableCheckbox={}}();
-//# sourceMappingURL=enableSelectAllTableCheckbox.js.map
\ No newline at end of file
+var DTFS_TFM;
```

Comment:
I'm not sure theres any changes here -- is it possible to revert these files (I'm also not sure whether itll fail due to the SHA not being the same -- I'm not sure how it handles line breaks)

Reply:
Good point, reverted these files.